corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -26,12 +27,15 @@
 from corehq.apps.userreports.exceptions import UserReportsError, ReportConfigurationNotFoundError
 from corehq.apps.userreports.models import get_report_config
 from corehq.apps.userreports.reports.data_source import ConfigurableReportDataSource
+from corehq.apps.userreports.tasks import compare_ucr_dbs
 from corehq.apps.app_manager.dbaccessors import (
     get_apps_in_domain, get_brief_apps_in_domain, get_apps_by_id
 )
 from six.moves import zip
 from six.moves import map
 
+UCR_COMPARISONS_THRESHOLD = 1000
```

Comment:
Will result in ~300 * # of UCRs selected to compare per day (300k restores per day)

Reply:
Not sure where you get that number from. The way I read this is that it will select 1 out of every 1000 to do the comparison. Considering that each restore has ~80 reports that comes to 1 in every ~12 restores which amounts to 16K per day (assuming 200K average restores per day).

Is it necessary to sample that many?

Reply:
It only does the comparison for reports that are in the dictionary `settings.UCR_COMPARISONS` so if we have 1 key/value pair there (which is the case atm since only one UCR has a custom implementation), then it would be:

(# of restores) / threshold * (# of UCR in this setting) * (# of times UCR is used in the app)

300,000 / 1,000 * 1 * 1 = 300 per day

I think that is report is only used once in the app, but could be wrong if it's used 2 or 3 times I think it should be fine since 900 queries would be tiny. Each of these should take < 1 second considering that ~80 reports takes ~50 seconds to generate during a restore and that includes all the time to convert them to XML

Reply:
:+1: thanks for clarifying